assessment_software/src/app/dialogue-box/successBox-component/successBox.component.ts

```diff
@@ -0,0 +1,10 @@
+import {Component, Input} from '@angular/core';
+
+@Component({selector:'app-success-box',
+templateUrl:'./successBox.component.html',
+styleUrls:['./successBox.component.css']})
+
+export class SuccessBoxComponent{
+    @Input() success_message:string='Successfully added assignment';
+    @Input() successBoxStyle:string='success';
```

Comment:
Currently, there is no way to remove the dialog box when the cancel button is pressed. Use the app-button-component API to implement that feature. 

In addition, when the 'view detail' button is pressed, we want to expose an API to the user to notify them that the button is pressed. this can be achieved using EventEmitter. It was done for app-button-component. Please review the app-button-component and add the same feature here for the dialogue. Also, after the button is pressed, the dialogue should hide/disappear.

Reply:
![dialogue box](https://github.com/Aben-Bel/assessment_software/assets/99379453/b9d2de28-ad16-4e6d-a6b9-313db9e37ba4)